Include/ComponentComposition/CameraComponent.h

```diff
@@ -4,20 +4,60 @@
 #include "Maths/Vectors.h"
 #include "Maths/Quaternion.h"
 #include "ComponentComposition/Component.h"
+#include "ComponentComposition/Color.h"
+#include "ComponentComposition/GameObject.h"
+#include "ComponentComposition/TransformComponent.h"
 
 namespace PlatinumEngine
 {
 	enum projectionType {PERSPECTIVE, ORTHOGRAPHIC};
+	enum clearFlagType {SKYBOX, BGCOLOR, DEPTH, NONE};
```

Comment:
Why we have SKYBOX mode in camera component? Maybe that is the responsibilty of sceneCamera? I'm not sure about that.

Reply:
I think it's fine. But the enum name should be capital, use enum class instead and lowercase for the values.

Reply:
Whoops...forgot to commit the changes earlier but they are up now